src/kvirc/ui/KviTreeWindowList.cpp

```diff
@@ -547,12 +557,15 @@ void KviTreeWindowListItemDelegate::paint(QPainter * p, const QStyleOptionViewIt
 			{
 				// The QWindowsXP style does not honor our colors. It uses the system ones instead.
 				// We can't accept it.
-				p->fillRect(option.rect,col);
+				QStyleOptionViewItemV4 opt4(option);
```

Comment:
I don't even know what the `opt4` var name referred to originally but for reasons of consistency I decided to stick to it.